Flutter/cartesian-charts/chart-types/spline-range-area-chart.md

```diff
@@ -68,62 +80,62 @@ The following code sample demonstrates how to set the [`splineType`](https://pub
 
 ## Border customization
 
-The borders of the spline range area chart can be customized using the [`borderDrawMode`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SplineRangeAreaSeries/borderDrawMode.html) property. The default value of the [`borderDrawMode`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SplineRangeAreaSeries/borderDrawMode.html) property is `all` and the other value is `excludeSides`.
+The borders of the spline range area chart can be customized using the [`borderDrawMode`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SplineRangeAreaSeries/borderDrawMode.html) property. The default value of the [`borderDrawMode`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SplineRangeAreaSeries/borderDrawMode.html) property is [`RangeAreaBorderMode.all`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RangeAreaBorderMode.html) and the other value is [`RangeAreaBorderMode.excludeSides`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RangeAreaBorderMode.html).
 
 {% highlight dart %} 
 
     @override
     Widget build(BuildContext context) {
         return Scaffold(
             body: Center(
-                child: Container(
                     child: SfCartesianChart(
-                        series: <CartesianSeries<SalesData, num>>[
-                            SplineRangeAreaSeries<SalesData, num>(
-                                borderDrawMode: RangeAreaBorderMode.all,
+                        series: <CartesianSeries<ChartData, num>>[
+                            SplineRangeAreaSeries<ChartData, num>(
+                                borderDrawMode:RangeAreaBorderMode.all,
+                                dataSource: chartData,
                                 borderWidth:2,
-                                borderColor: Colors.red
+                                borderColor: Colors.red,
+                                xValueMapper: (ChartData sales, _) => sales.xValue,
+                                lowValueMapper: (ChartData sales, _) => sales.lowValue,
+                                highValueMapper: (ChartData sales, _) => sales.highValue
                             )
                         ]
                     )
                 )
-            )
-        );
-    }
+            );
+        }
 
 {% endhighlight %}
 
 ![splinerangearea_bordermode](cartesian-chart-types-images/splinerangearea_bordermode.png)
 
-## See Also
+#### See Also
 
-* [Color palette](./series-customization#color-palette) 
-* [Color mapping](./series-customization#color-mapping-for-data-points)
-* [Animation](./series-customization#animation)
-* [Gradient](./series-customization#gradient-fill)
-* [Empty points](./series-customization#empty-points)  
-* [Sorting](./series-customization##sorting) 
+* [color palette](https://help.syncfusion.com/flutter/cartesian-charts/series-customization#color-palette) 
```

Comment:
why do we need to give this full link, I already sked to give proper link

Reply:
I have changed the link to below, also the error was occur
![image](https://user-images.githubusercontent.com/79507200/155924810-14e392d6-5096-4930-ad4f-661983f158f3.png)